app/src/main/res/values/styles.xml

```diff
@@ -110,4 +110,15 @@
         <item name="android:lineSpacingExtra">12sp</item>
         <item name="android:textStyle">normal</item>
     </style>
+
+    <!-- Button style -->
+    <style name="Button_style" parent="@android:style/TextAppearance.Small">
+        <item name="android:width">312dp</item>
```

Comment:
to add the effect you were mentioning: android:foreground="?selectableItemBackground" or ?selectableItemBackgroundBorderless, check which one looks better